src/EnergyPlus/Material.cc

```diff
@@ -291,14 +293,15 @@ void GetMaterialData(EnergyPlusData &state, bool &ErrorsFound) // set to true if
     // Add the 6" heavy concrete for constructions defined with F or C factor method
     if (TotFfactorConstructs + TotCfactorConstructs >= 1) {
         ++MaterNum;
-        auto *thisMaterial = state.dataMaterial->Material(MaterNum);
+        auto *thisMaterial = new Material::MaterialChild;
+        state.dataMaterial->Material(MaterNum) = thisMaterial;
         thisMaterial->Group = MaterialGroup::RegularMaterial;
         thisMaterial->Name = "~FC_Concrete";
         thisMaterial->Thickness = 0.15;    // m, 0.15m = 6 inches
         thisMaterial->Conductivity = 1.95; // W/mK
         thisMaterial->Density = 2240.0;    // kg/m3
         thisMaterial->SpecHeat = 900.0;    // J/kgK
-        thisMaterial->Roughness = DataSurfaces::SurfaceRoughness::MediumRough;
+        thisMaterial->Roughness = Material::SurfaceRoughness::MediumRough;
```

Comment:
Don't need `Material::` namespace scoping when you are inside that namespace.

Reply:
right right